facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -121,9 +121,9 @@ object FrontChecks {
 
 }
 
-class FaciaPicker extends GuLogging {
+object FaciaPicker extends GuLogging {
```

Comment:
This was to make it the same as [ArticlePicker](https://github.com/guardian/frontend/blob/main/article/app/services/dotcomrendering/ArticlePicker.scala#L27) otherwise it leaves future us wondering why it might have been done differently - and it doesn't need to be a class as it's never instantiated.